src/actions/tripRequestAction.js

```diff
@@ -30,6 +30,33 @@ export const requestTrip = data => async dispatch => {
 		});
 };
 
+export const editTripRequest = (data, tripId) => async dispatch => {
+	dispatch({ type: 'LOADING', payload: true });
+	return await axios
+		.patch(`${process.env.BACKEND_BASE_URL}/api/v1/trips/${tripId}`, data, {
+			headers,
+		})
+		.then(response => {
+			const request = response.data.data;
```

Comment:
@Jajabenit250 are you really getting a request or the trip_request? The variable name is kind misleading.